.github/actions/environment/action.yml

```diff
@@ -44,17 +47,4 @@ runs:
           maui-android \
           ${{ runner.os == 'macOS' && 'maui-ios maui-maccatalyst maui-windows' || '' }} \
           ${{ runner.os == 'Windows' && 'maui-windows' || '' }} \
-          --temp-dir "${{ runner.temp }}" --from-rollback-file rollback.json
-
-    - name: Restore workloads
-      shell: bash
-      run: dotnet workload restore
-
-    - name: Update workloads
-      shell: bash
-      run: dotnet workload update
-
-    - name: Log what we have in the workloads
-      shell: bash
-      run: dotnet workload update
-      
\ No newline at end of file
+          --temp-dir "${{ runner.temp }}"
```

Comment:
Nice! We don't need rollback.json anymore?

Reply:
Yess!